cmd/triggers/generate_test.go

```diff
@@ -54,7 +55,7 @@ func Test_TriggerGenerate_accept_prompt(t *testing.T) {
 	}
 
 	t.Run(tt.name, func(t *testing.T) {
-		ctx, clientsMock := prepareMocks(tt.triggersListResponse, tt.globResponse, tt.triggersCreateResponse, nil /*trigger create error*/)
+		ctx, clientsMock := prepareMocks(t, tt.triggersListResponse, tt.globResponse, tt.triggersCreateResponse, nil /*trigger create error*/)
```

Comment:
note: We need `t *testing.T` in order to called `slackcontext.MockContext(t.Context())`